### DISCUSSION TITLE
Can't use static files if I use evenlet/gevent with Django?

### DISCUSSION THREAD

**Discussion by carlosevi94** [miguelgrinberg/python-socketio#949] [Q&A] [answered]
Hello everyone!. 

I am using python-socketio to develop a small online game, and I have decided that for convenience I will use django for the data models and ORM. 

I was having trouble detecting the disconnect event, and after reading some posts [(754)](https://github.com/miguelgrinberg/python-socketio/discussions/754) I discovered that the problem was that I wasn't using the proper configuration to use sockets (previously I was just using the example configuration from the django example). 

I decide to install eventlet, and configure it.

```python
# on wsgi.py: 

django_app = get_wsgi_application()
application = socketio.WSGIApp(sio, django_app)

import eventlet
eventlet.wsgi.server(eventlet.listen(('', 8000)), application)


```
After installing eventlet I got the disconnect event detected correctly, but now no static files (js scripts, css files, images, etc) are loaded.

I've been looking for information but haven't found anything. Maybe I don't know what I'm looking for exactly. 
I would copy the error traces from the server, but it just shows a `Not Found: /static/css/my_styles.css` type message. If I launch the application without eventlet, the statics are displayed correctly. 

Could you give me some advice or tips to fix this? 

Regards, great job with this library, its amaizing.

**Comment by miguelgrinberg** [comment, marked as answer]
See https://github.com/miguelgrinberg/python-socketio/issues/943

**Reply by carlosevi94** [reply]
Yep, that solved my problem too.

I didn't see that issue, I look up directly on discussion, my bad. 

Thanks for answer so fast!! :smile:
